### ISSUE TITLE
`logByType` issue in IE10

### ISSUE THREAD

**Issue by alex-phillips** [videojs/video.js#3442]
## Description
I've only noticed this in IE10 (works fine in IE9 and IE11). In the function `logByType` (in `log.js` src file), the following error is thrown:

```
SCRIPT65535: Invalid calling object
```

If you simply surround the `fn(args.map....` call in a try/catch, it solves the issue. However, I don't know if this is the best solution.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.

1. Simply run the code in IE10. Although, I was also using the VAST/VPAID plugin.

### Expected
No exception to be thrown and VideoJS initialize normally.

### Actual
Exception thrown

### Error output
```
SCRIPT65535: Invalid calling object
```

#### videojs
Current `master` branch
#### browsers
IE10
#### OSes
Windows *
### plugins
VAST/VPAID plugin

**Comment by misteroneill** [comment]
Confirmed this and will fix - thanks for catching it @alex-phillips

**Comment by alex-phillips** [comment]
@misteroneill I can quick submit a PR for the try/catch solution if that is preferable to get it merged in faster. Just let me know. Thanks!

**Comment by misteroneill** [comment]
@alex-phillips Thanks, but I think I've got something. Will have a PR shortly. 😄

**Comment by gkatsev** [comment]
Should be fixed in 5.11.0.
